info/v2/container.go

```diff
@@ -52,6 +52,14 @@ type MemorySpec struct {
 	SwapLimit uint64 `json:"swap_limit,omitempty"`
 }
 
+type ContainerInfo struct {
+	// Describes the container.
+	Spec ContainerSpec `json:"spec,omitempty"`
```

Comment:
> I really wish we separate the internal API from the wire format soon.

Agreed. I think [this issue](https://github.com/google/cadvisor/issues/606) is asking for that? I'll try to prioritize it after the 1.2 freeze.